BuildAssets/test-linux-package.sh

```diff
@@ -15,6 +15,14 @@ if [[ "$OSRELID" == "rhel" && ( -z "$REDHAT_SUBSCRIPTION_USERNAME" || -z "$REDHA
   exit 1
 fi
 
+if [[ "$OSRELID" == "debian" ]]; then
+  OSRELVERSIONID="$(. /etc/os-release && echo $VERSION_ID)"
+  if [[ "$OSRELVERSIONID" == "9" ]]; then
+    # from https://unix.stackexchange.com/a/743865
+    echo "deb http://archive.debian.org/debian stretch main" > /etc/apt/sources.list
+  fi
+fi
```

Comment:
this was supposed to fix the apt-get update problem but it didn't seem to work, leaving it here for future reference